api/v1/routes/blog.py

```diff
@@ -3,15 +3,29 @@
 from sqlalchemy.orm import Session
 from api.db.database import get_db
 from api.v1.models.user import User
-from api.utils.dependencies import get_current_user
+from api.utils.dependencies import get_current_user, get_super_admin
 from api.v1.services.blog import BlogService
 from fastapi import APIRouter, Depends, HTTPException, status
 from typing import List
 from api.v1.models.blog import Blog
 from api.v1.schemas.blog import BlogResponse
+from api.v1.schemas.blog import BlogCreate
```

Comment:
change this

from api.v1.schemas.blog import BlogResponse
from api.v1.schemas.blog import BlogCreate

to 

from api.v1.schemas.blog import BlogResponse, BlogCreate